lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -41,7 +41,6 @@ using magma::sctpd::SctpdUplinkClient;
 int signalMask(void) {
   sigset_t set;
   sigemptyset(&set);
-  sigaddset(&set, SIGSEGV);
```

Comment:
We should also remove this from line 61 below. Further, since signalMask is already called at the beginning of `main()`, we can get rid of the sigprocmask call on line 65.

Reply:
Deleted them too.